src/pages/getting-started/app-guides/app-authorization/index.mdx

```diff
@@ -7,12 +7,30 @@ tagline: "Find out how to implement an authorization flow and configure the App
 desc: "Find out how to implement an authorization flow and configure the App authorization building block."
 ---
 
+# Authorization for LiveChat apps
+
 If you want to build an integration that calls the LiveChat APIs, you'll need to implement an authorization flow and configure the **App authorization** building block.
 
 - <SectionLink to={"/authorization/authorizing-api-calls/"}>Read about different authorization flows</SectionLink>
 
 - <SectionLink to={"/authorization/authorization-in-practice/"}>Tutorial: Authorization in practice </SectionLink>
 
 <div style={{maxWidth: '600px'}}>
-  <img src="/images/getting-started/app-guides/app-authorization/livechat-authorization-block.png" alt="Authorizaton building block" width="845px" height="385px"/>
+  <img src="/images/getting-started/app-guides/app-authorization/livechat-authorization-block.png" alt="Authorizaton building block" width="52px" height="53px"/>
 </div>
+
+
+# Authorization for HelpDesk apps
+
+It is OAuth 2.1 that is used to securely access the data of the HelpDesk users.
+
+[HelpDesk API](https://api.labs.helpdesk.com/docs#section/Authentication) shares an authentication and authorization system with LiveChat. If you want to build an integration within HelpDesk, you'll need to configure the App authorization building block and authenticate or authorize within HelpDesk.
+
+To read more about authenticating or authorizing within HelpDesk, take a look at:
+
+- [Agent Authorization](/authorization/agent-authorization).
+- [Sign in with LiveChat flow](/authorization/sign-in-with-livechat).
+
+<div style={{maxWidth: '600px'}}>
```

Comment:
```suggestion
<div style={{maxWidth: '200px'}}>
```

Reply:
w sumie to nie jestem pewna czy tu jest potrzebne to zdj jak jest takie samo?

Reply:
No i właśnie miałam z tym zagwozdkę...bo z drugiej strony jak dam tylko przy jednym to ewidentnie 'brakuje' przy drugim...finalnie stwierdziłam, że lepiej wygląda jak jest przy obu. Ale najwyżej zobaczę co Ola powie jeszcze i będzie akurat głos rozstrzygający xd